lib/Mailer/Methods/ErrorMappers/MailPoetMapper.php

```diff
@@ -91,29 +91,17 @@ private function getSubscribersErrors($result_parsed, $subscribers) {
 
   private function getUnauthorizedEmailMessage($sender, $newsletter) {
     $email = $sender ? $sender['from_email'] : null;
-    if ($email && (new FreeDomains())->isEmailOnFreeDomain($email)) {
-      $message = '<p>' . sprintf(WPFunctions::get()->__('The MailPoet Sending Service can’t send email with the email address <i>%s</i>. You need to use an address like <i>‌‌‌‌‌‌‌‌‌‌‌‌‌‌‌‌‌‌you@yourdomain.com</i>.', 'mailpoet'), $email) . '</p>';
-      $message .= '<p>';
-      if ($newsletter && $newsletter['id']) {
-        $message .= '<a class="button button-primary" href="admin.php?page=mailpoet-newsletters#/send/' . $newsletter['id'] .'">';
-      } else {
-        $message .= '<a class="button button-primary" href="admin.php?page=mailpoet-settings">';
-      }
-      $message .= WPFunctions::get()->__('Change my email address', 'mailpoet');
-      $message .= '</a>';
-    } else {
-      $message = sprintf(WPFunctions::get()->__('<p>The MailPoet Sending Service did not send your latest email because the address <i>%s</i> is not yet authorized.</p>', 'mailpoet'), $email ?: WPFunctions::get()->__('Unknown address'));
-      $message .= '<p>';
-      $message .= Helpers::replaceLinkTags(
-        WPFunctions::get()->__('[link]Authorize your email in your account now.[/link]', 'mailpoet'),
-        'https://account.mailpoet.com/authorization',
-        array(
-          'class' => 'button button-primary',
-          'target' => '_blank',
-          'rel' => 'noopener noreferrer',
-        )
-      );
-    }
+    $message = sprintf(WPFunctions::get()->__('<p>The MailPoet Sending Service did not send your latest email because the address <i>%s</i> is not yet authorized.</p>', 'mailpoet'), $email ?: WPFunctions::get()->__('Unknown address'));
```

Comment:
Could you please move paragraph tags out of the translation? I'm afraid that some translator might accidentally remove it and break formatting.